cranelift/filetests/filetests/vcode/x64/basic.clif

```diff
@@ -0,0 +1,16 @@
+test compile
+target x86_64
+feature "experimental_x64"
+
+function %f(i32, i32) -> i32 {
+block0(v0: i32, v1: i32):
+    ; check: pushq   %rbp
+    ; check: movq    %rsp, %rbp
+    v2 = iadd v0, v1
+    ; check: addl    %esi, %edi
+    return v2
+    ; check: movq    %rdi, %rax
+    ; check: movq    %rbp, %rsp
+    ; check: popq    %rbp
+    ; check: ret
+}
```

Comment:
I would have expected these tests under `.../filetests/isa/x64`?

Reply:
And looking at the `filetest` structure more closely, what do you think about a slight re-organization in a separate PR:
 - `filetests/vcode/*` get moved under `filetests/isa`
 - move a bunch of the optimization directories under a new `filetests/opt`
I'm just thinking that finding where to put new tests is getting challenging.

Reply:
I've moved them over to isa/x64; for the rest, I'm not sure what you mean by the optimization directories, so we could discuss it somewhere convenient.